pyproject.toml

```diff
@@ -60,7 +60,6 @@ test = [
   "flake8-docstrings",
   "hacking >= 1.0",
   "pysatSpaceWeather",
-  "pytest",
```

Comment:
why is this removed?

Reply:
Because it is a part of the pysat requirements.  No need to request it twice.